backend/bin/deal-observer-backend.js

```diff
@@ -63,6 +63,32 @@ const dealObserverLoop = async (makeRpcRequest, pgPool) => {
   }
 }
 
+export const pieceIndexerLoop = async (makeRpcRequest, makePixRequest, pgPool) => {
```

Comment:
I believe this function is never called

Reply:
IIRC, @NikolasHaimerl mentioned in an older comment that he does not want to run it yet, but I don't remember details.

I see you already added a call of this function at the bottom of this file, so I think this comment is resolved now?

```js
await Promise.all([
  // TODO: Define `pixRequest`
  pieceIndexerLoop(rpcRequest, () => {}, pgPool),
  observeActorEventsLoop(rpcRequest, pgPool),
  sparkApiSubmitDealsLoop(pgPool, {
    sparkApiBaseUrl: SPARK_API_BASE_URL,
    // ...
  })
])
```

Reply:
just need to fix `pixRequest`, but yes 👍